src/VolleyManagement.Backend/VolleyManagement.UI/Infrastructure/IOCUIModule.cs

```diff
@@ -24,11 +24,7 @@ public void RegisterDependencies(IIocContainer container)
             {
                 container.Register<IMailService, DebugMailService>(IocLifetimeEnum.Scoped);
             }
-            else if (Is<IISDeployment>.Enabled)
-            {
-                container.Register<IMailService, GmailAccountMailService>(IocLifetimeEnum.Scoped);
-            }
-            else
+            else if (Is<IISDeployment>.Disabled)
```

Comment:
Now we might have potential problem. If either IISDeployment and DebugMode flags are not set properly we won't have IMailService registered and app will fail.

Make sure DebugMailService is registered if nothing else is registered.

Reply:
![default](https://user-images.githubusercontent.com/11539042/36209153-a69c5b40-11a3-11e8-966c-ae97b4a3895b.PNG)

Reply:
Do you mean this?

Reply:
yes